### PR TITLE
asciinema2gif 0.2 (new formula)

### PR DESCRIPTION
**Pull request by vitorgalvao**
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

### New Formulae Submissions:

- [x] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

File: Library/Formula/asciinema2gif.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  sha256 "fa5834aaec162b992768bed854d69eecff0862c733d4be0334dbc3b9fe86481c"
+  head "https://github.com/tav/asciinema2gif.git"
+
+  depends_on "gifsicle"
+  depends_on "imagemagick"
+  depends_on "phantomjs"
+
+  def install
+    bin.install "asciinema2gif"
+    bin.install "render.js"
```
Comment (vitorgalvao) [inline Library/Formula/asciinema2gif.rb]: I have a question here, though. `render.js` needs to be in the same directory as `asciinema2gif`, * but* it does need to be symlinked, as it should never be called directly (it’s called by `asciinema2gif`). Is there a standard equivalent way to `bin.install` to move to the `bin` directory without symlinkg?

**Comment by vitorgalvao** [comment]
Not sure what’s failing the build.

```bash
$ brew audit --strict --online ./asciinema2gif.rb
==> brew style asciinema2gif

1 file inspected, no offenses detected
$ brew install ./asciinema2gif.rb
==> Downloading https://github.com/tav/asciinema2gif/archive/0.2.tar.gz
Already downloaded: /Library/Caches/Homebrew/asciinema2gif-0.2.tar.gz
🍺  /usr/local/Cellar/asciinema2gif/0.2: 4 files, 9.3K, built in 0 seconds
```

File: Library/Formula/asciinema2gif.rb (hunk 1 above)
Comment (vitorgalvao) [inline Library/Formula/asciinema2gif.rb]: This seems to be what’s failing the build, though it doesn’t do so locally.

```bash
$ brew audit --strict --online ./asciinema2gif.rb
==> brew style asciinema2gif

1 file inspected, no offenses detected
$ brew install ./asciinema2gif.rb
==> Downloading https://github.com/tav/asciinema2gif/archive/0.2.tar.gz
Already downloaded: /Library/Caches/Homebrew/asciinema2gif-0.2.tar.gz
🍺  /usr/local/Cellar/asciinema2gif/0.2: 4 files, 9.3K, built in 0 seconds
```

Comment (DomT4) [inline Library/Formula/asciinema2gif.rb]: It sounds like you want:
```ruby
(libexec/"bin").install %w[asciinema2gif render.js]
bin.write_exec_script libexec/"bin/asciinema2gif"
```

Comment (vitorgalvao) [inline Library/Formula/asciinema2gif.rb]: Looks like it. Updated. Thank you.

**Comment by wong2** [comment]
thanks a lot!

**Comment by DomT4** [comment]
Should this perhaps have a recommended `asciinema` dependency or can you use it without `asciinema`?

**Comment by vitorgalvao** [comment]
@DomT4 You can use it without. You basically give `asciinesa2gif` the URL for any asciicast and it converts it to a gif. You don’t need `asciinema` at all, you can use it just as well on other people’s online asciicasts.

**Comment by DomT4** [comment]
Cool, thanks for the quick explanation! Merged in https://github.com/Homebrew/homebrew/commit/3ca7883123075d5c02ef7fbe16cb1f67b96aaa54. Thank you @vitorgalvao :smiley:

**Comment by vitorgalvao** [comment]
Thank you for the help and the merge.

**Pull request closed without merging**